policykit/policyengine/module.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
+import json
+from policyengine.models import (
+    CommunityPlatform,
+    CustomAction,
+    Procedure,
+    Execution,
+)
+
+class PolicyFactory():
```

Comment:
shouldn't the policy factory be creating policies?

Reply:
Yes, indeed. I will pick a better name here

Reply:
well we do need a place to create Policy objects from these template classes, no?

Reply:
I plan to use PolicyFactory for my creating these templates and putting the logic of creating policies from these templates in the `views.py` because it involves the interaction with the front end.